src/css/application.css

```diff
@@ -266,6 +266,23 @@ body {
   background-color: var(--color-gray);
 }
 
+.librarypicker__item.dashboard__menu-item_active::before {
+    content: '';
+    position: absolute;
+    border-color: #000;
+    border-style: solid;
+    border-width: 0 0.15rem 0.15rem 0;
```

Comment:
I really like the way this looks, although when I originally suggested using a `:before` pseudo element, I assumed you’d just use the unicode checkbox in the `content` property. I’m totally down with what you’ve got here but also curious as to whether you tried the unicode option and found it wanting?